wdl/MainVcfQc.wdl

```diff
@@ -0,0 +1,843 @@
+    for tarball_fname in ~{sep=" " plot_qc_site_level_external_benchmarking_tarballs}; do
+      dname="$( basename -s '.tar.gz' $tarball_fname )_site_level_benchmarking_plots/"
+      mkdir ~{prefix}_SV_VCF_QC_output/plots/supplementary_plots/$dname
+    done
+    mkdir ~{prefix}_SV_VCF_QC_output/plots/supplementary_plots/per_sample_plots/
+    if ~{defined(plot_qc_per_family_tarball)}; then
+      mkdir ~{prefix}_SV_VCF_QC_output/plots/supplementary_plots/sv_inheritance_plots/
+    fi
+    for tarball_fname in ~{sep=" " plot_qc_per_sample_external_benchmarking_tarballs}; do
+      dname="$( basename -s '.tar.gz' $tarball_fname )_per_sample_benchmarking_plots/"
+      mkdir ~{prefix}_SV_VCF_QC_output/plots/supplementary_plots/$dname
+    done
+
+    # Process VCF-wide stats
+    cp ~{vcf_stats} \
+      ~{prefix}_SV_VCF_QC_output/data/~{prefix}.VCF_sites.stats.bed.gz
+    cp ~{vcf_stats_idx} \
+      ~{prefix}_SV_VCF_QC_output/data/~{prefix}.VCF_sites.stats.bed.gz.tbi
+
+    # Process VCF-wide plots
+    tar -xzvf ~{plot_qc_vcfwide_tarball}
+    cp plotQC_vcfwide_output/main_plots/* \
+      ~{prefix}_SV_VCF_QC_output/plots/main_plots/
+    cp plotQC_vcfwide_output/supporting_plots/vcf_summary_plots/* \
+      ~{prefix}_SV_VCF_QC_output/plots/supplementary_plots/vcf_summary_plots/
+
+    # Process site-level external benchmarking plots
+    if ~{defined(plot_qc_site_level_external_benchmarking_tarballs)}; then
+      # For now, just dump them all into a tmp holding directory
+      # TODO: clean this up so it appropriately relocates all files & plots
+      cp ~{sep=" " plot_qc_site_level_external_benchmarking_tarballs} \
+        ~{prefix}_SV_VCF_QC_output/plots/supplementary_plots/external_benchmarking_tarballs/
+    fi
+
+    # Process per-sample stats
+    tar -xzvf ~{collect_qc_per_sample_tarball}
+    cp ~{prefix}_perSample_VIDs_merged/*.VIDs_genotypes.txt.gz \
+      ~{prefix}_SV_VCF_QC_output/data/variant_info_per_sample/
+
+    # Process per-sample plots
+    tar -xzvf ~{plot_qc_per_sample_tarball}
+    cp ~{prefix}_perSample_plots/main_plots/* \
+      ~{prefix}_SV_VCF_QC_output/plots/main_plots/
+    cp ~{prefix}_perSample_plots/supporting_plots/per_sample_plots/* \
+      ~{prefix}_SV_VCF_QC_output/plots/supplementary_plots/per_sample_plots/
+
+    # Process per-family plots
+    if ~{defined(plot_qc_per_family_tarball)}; then
+      tar -xzvf ~{plot_qc_per_family_tarball}
+      cp ~{prefix}_perFamily_plots/main_plots/* \
+        ~{prefix}_SV_VCF_QC_output/plots/main_plots/ || true
+      cp ~{prefix}_perFamily_plots/supporting_plots/sv_inheritance_plots/* \
+        ~{prefix}_SV_VCF_QC_output/plots/supplementary_plots/sv_inheritance_plots/ || true
+    fi
+
+    # Process per-sample external benchmarking plots
+    if ~{defined(plot_qc_per_sample_external_benchmarking_tarballs)}; then
+      # For now, just dump them all into a tmp holding directory
+      # TODO: clean this up so it appropriately relocates all files & plots
```

Comment:
Does this `TODO` (and the other one earlier in the task) need immediate action or does it work well enough to merge for now and come back later?